libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -342,7 +358,7 @@ AP_GPS_UBLOX::read(void)
                 _delay_time = 750;
             }
         } else {
-            _delay_time = 2000;
+            _delay_time = 4000;
```

Comment:
more or less yes. some of the correction data will come in as 2 mavlink packets. because the packet is over 110 bytes